src/modules/_nomodule/PixelManipulation.js

```diff
@@ -47,25 +47,53 @@ module.exports = function PixelManipulation(image, options) {
       /* Allows for Flexibility
        if per pixel manipulation is not required */
 
-      for (var x = 0; x < pixels.shape[0]; x++) {
-        for (var y = 0; y < pixels.shape[1]; y++) {
-          let pixel = options.changePixel(
-            pixels.get(x, y, 0),
-            pixels.get(x, y, 1),
-            pixels.get(x, y, 2),
-            pixels.get(x, y, 3),
-            x,
-            y
-          );
-
-          pixels.set(x, y, 0, pixel[0]);
-          pixels.set(x, y, 1, pixel[1]);
-          pixels.set(x, y, 2, pixel[2]);
-          pixels.set(x, y, 3, pixel[3]);
-
-          if (!options.inBrowser && !process.env.TEST) pace.op();
+      const imports = {
+        env:{
+          // paceOp: require('pace').op,
+          consoleLog:console.log,
+          perform: function(x,y){
+            let pixel = options.changePixel(
+              pixels.get(x, y, 0),
+              pixels.get(x, y, 1),
+              pixels.get(x, y, 2),
+              pixels.get(x, y, 3),
+              x,
+              y
+            );
+        
+            pixels.set(x, y, 0, pixel[0]);
+            pixels.set(x, y, 1, pixel[1]);
+            pixels.set(x, y, 2, pixel[2]);
+            pixels.set(x, y, 3, pixel[3]);
+          }
         }
-      }
+      };
+      const inBrowser = (options.inBrowser)?1:0;
+      const test = (process.env.TEST)?1:0;
+      WebAssembly.instantiateStreaming(fetch('./program.wasm'), imports)
```

Comment:
Ah, you're using `fetch` here instead of `fs`? Or, where was that code you cited before? 

```
if (options.changePixel) {
      isDone = true;
      /* Allows for Flexibility
       if per pixel manipulation is not required */
      const fs = require('fs');
      console.log(fs.readFileSync)
      var source = fs.readFileSync('./p1.wasm');
      var typedArray = new Uint8Array(source);
```
Please help us follow along so we can help out! Thanks!!!